modin/pandas/test/test_series.py

```diff
@@ -2204,12 +2204,13 @@ def test_loc(data):
     data = np.arange(100)
     modin_series = pd.Series(data, index=index).sort_index()
     pandas_series = pandas.Series(data, index=index).sort_index()
+    # Using 'fmt: skip' below as 'black' and 'flake8' can't agree on how this should be formatted
     modin_result = modin_series.loc[
         (slice(None), 1),
-    ]
+    ]  # fmt: skip
     pandas_result = pandas_series.loc[
         (slice(None), 1),
-    ]
+    ]  # fmt: skip
```

Comment:
`black` wants this to be formatted exactly as:
```python
modin_results = modin_series.loc[(slice(None), 1),]
```
when `flake8` agrees on either of these:
```python
modin_results = modin_series.loc[(slice(None), 1), ]
# or
modin_results = modin_series.loc[
    (slice(None), 1),
]
```
so let's just keep the old formatting

Reply:
and yes, the comma is mandatory here as it decides whether the `.__getitem__` receives a single tuple or a nested one:
```python
>>> pandas_series.loc[(slice(None), 1)]
f
0     1
1    11
...
>>> pandas_series.loc[(slice(None), 1),]
f  s
0  1     1
1  1    11
...
```